harness/determined/gpu.py

```diff
@@ -48,12 +61,13 @@ def get_gpus() -> List[GPU]:
                     GPU(
                         id=int(fields["index"]),
                         uuid=fields["uuid"].strip(),
-                        load=float(fields["utilization.gpu"]) / 100,
-                        memoryUtil=float(fields["memory.used"]) / float(fields["memory.total"]),
+                        load=float_or_default(fields, "utilization.gpu", 0.0) / 100,
```

Comment:
Is a partial GPU one item in the list or one whole GPU is one item in the list?

Reply:
Each partial GPU is one item in the list. As far as Determined can tell, a partial GPU is a whole GPU, except that it will end up having fewer compute resources, less memory, and you won't (in Kubernetes at least) have the ability to look up utilization.

Reply:
Just to be clear, though, NVIDIA does not recommend using multipe partial GPUs for a single task. I tested using distributed training with multiple partial GPUs, and it works, but the recommend configuration would be to just use a whole GPU in that case. I can see MIG being used for a massive hyperparameter search with single-slot trials, or perhaps an inference-heavy workload. I don't think it will be useful for most distributed training.